test/generators/erb_generator_test.rb

```diff
@@ -40,4 +40,10 @@ def test_component_with_namespace_and_sidecar
 
     assert_file "app/components/admins/user_component/user_component.html.erb"
   end
+
+  def test_component_with_inline
```

Comment:
I think it more logical to have this test here instead of inside `ComponentGeneratorTest`.